data/angle/angle.yml

```diff
@@ -0,0 +1,6 @@
+name: angle
```

Comment:
Your function name is vector_angle

Reply:
I dont think this is a problem.
The function name can be different from the file name. 
The file name should appear in the yml file. 
See cubes.

Reply:
Thanks for clarifying @navneeth3005